services/api/src/domain/player.ts

```diff
@@ -13,6 +13,7 @@ export interface Player {
   id: string;
   name: string;
   color: PlayerAvatarColor;
+  avatar: string;
```

Comment:
is this mandatory or optional?

Reply:
I would say it is mandatory. If the user does not have custom avatar, its gonna be '' (empty string)